src/pocketmine/network/mcpe/protocol/types/DeviceIds.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\network\mcpe\protocol\types;
+
+interface DeviceIds{
+
+	public const UNKNOWN = -1;
+	public const ANDROID = 1;
+	public const IOS = 2;
+	public const OSX = 3;
+	public const AMAZON = 4;
+	public const GEAR_VR = 5;
+	public const HOLOLENS = 6;
+	public const WINDOWS_10 = 7;
+	public const WIN32 = 8;
+	public const DEDICATED = 9;
+	public const TVOS = 10;
+	public const PLAYSTATION = 11;
```

Comment:
Isn't play station two words?

Reply:
I don't think anyone is going to think to use `PLAY_STATION`. My first thought would be `PS4`.